cpp/src/slice2cpp/Gen.cpp

```diff
@@ -72,12 +72,12 @@ namespace
         }
     }
 
-    string getDeprecateSymbol(const ContainedPtr& p1)
+    string getDeprecatedSymbol(const ContainedPtr& p1)
     {
         string deprecatedSymbol;
-        if (p1->isDeprecated(true))
+        if (p1->isDeprecated(false))
```

Comment:
This bool is `checkParent`. It used to be true, because we supported
`deprecating an interface` == `deprecate it's operations`.
Now that we no longer use this logic, it was switched to false.

Reply:
I would add a comment for readability.

Reply:
Added!